tests/errors/semantic/non_blocking/wrong_class_type_annotation.py

```diff
@@ -11,4 +11,3 @@ def get_4(self : 'A'):
 if __name__ == '__main__':
     myA : int = A()
 
-    print(myA.x)
```

Comment:
`myA` now initialised as `int` so `x` not found leading to blocking error. This line is removed as it is not what we want to test in this file